src/deploy/osp_deployer/settings/config.py

```diff
@@ -572,6 +572,18 @@ def __init__(self, settings_file):
                 except AttributeError:
                     node.skip_raid_config = False
                     pass
+                try:
+                    if node.skip_bios_config == "true":
+                        node.skip_bios_config = True
+                except AttributeError:
+                    node.skip_bios_config = False
+                    pass
+                try:
+                    if node.skip_nic_config == "true":
+                        node.skip_nic_config = True
+                except AttributeError:
+                    node.skip_nic_config = False
+                    pass
 
```

Comment:
Might be nice to introduce the standard python library distutils to our stuff. Then we could simplify above into something like: node.skip_bios_config = bool(util.strtobool(node.skip_bios_config))
https://docs.python.org/2/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool

Would allow us to skip all the exception handling etc I believe.